src/decision/dMCDA.jl

```diff
@@ -689,16 +689,16 @@ function run_site_selection(dom::Domain, scenarios::DataFrame, sum_cover::Abstra
 
     n_sites = length(dom.site_ids)
```

Comment:
I think this wasn't caused by this commit BUT this variable seems no to be used.